packages/ra-core/src/actions/accumulateActions.ts

```diff
@@ -36,8 +36,8 @@ export interface CrudGetMatchingAccumulateAction {
 export const crudGetMatchingAccumulate = (
     reference: string,
     relatedTo: string,
-    pagination: object,
-    sort: object,
+    pagination: { page: number; perPage: number },
```

Comment:
We should probably extract this into a dedicated type or interface as it is used in other parts of the project